test/functional/neutronless/disconnected_service/test_disconnected_service_creation.py

```diff
@@ -421,18 +403,15 @@ def test_nosegid_listener(setup_l2adjacent_test, bigip):
     assert ERROR_MSG_MISCONFIG not in open(logfilename).read()
     create_uris = (set(after_create_registry.keys()) -
                    set(start_registry.keys()))
-    assert create_uris == (SEG_INDEPENDENT_LB_URIS | NOSEG_LISTENER_URIS |
-                           NOSEG_LB_URIS)
+    assert create_uris == (SEG_INDEPENDENT_LB_URIS)
+
     rpc = icontroldriver.plugin_rpc
     LOG.debug(rpc.method_calls)
-    assert rpc.update_listener_status.call_args_list == [
-        call(u'105a227a-cdbf-4ce3-844c-9ebedec849e9', 'ACTIVE', 'OFFLINE')
-    ]
-    assert rpc.update_loadbalancer_status.call_args_list == [
-        call(u'50c5d54a-5a9e-4a80-9e74-8400a461a077', 'ACTIVE', 'OFFLINE')
-    ]
+    assert not rpc.update_listener_status.called
```

Comment:
Will the polling request occur in the plugin_rpc module? If so, can we wait for it, then ensure we made that call correctly?